tests/TestCase/View/StringTemplateTest.php

```diff
@@ -80,6 +80,27 @@ public function testAddWithInvalidTemplate(): void
         ];
         $this->expectException(InvalidArgumentException::class);
         $this->template->add($templates);
+
+        $templates = [
+            'link' => '<a href="{{url}}">{{text}}</a>',
+            'invalid' => false,
+        ];
+        $this->expectException(InvalidArgumentException::class);
+        $this->template->add($templates);
```

Comment:
How are these tests related to your changes?

Reply:
It was related to the other pull request. It checks for more then just a null template. I removed it from here.